datafusion/proto/proto/datafusion.proto

```diff
@@ -739,6 +739,13 @@ message IntervalMonthDayNanoValue {
   int64 nanos = 3;
 }
 
+message StructValue {
+  // encode null explicitly to distinguish a struct with no fields (is
```

Comment:
If there are no fields there can't be any values, null or otherwise? I therefore think we could drop this?

Reply:
I will try and remove it

Reply:
Added an assert that there are always at least one `Field` / value and removed explicit support is_null in 88370bf7f